alembic/operations/ops.py

```diff
@@ -1108,7 +1114,12 @@ def from_table(cls, table, _namespace_metadata=None):
         return cls(
             table.name,
             schema=table.schema,
-            table_kw=table.kwargs,
+            table_kw={
+                "comment": table.comment,
+                "info": table.info,
+                "prefixes": table._prefixes,
+                **table.kwargs,
+            },
```

Comment:
This is the only way I can see to not break the class' `__init__()` public API.

I cannot help but wonder how much of public API this is though. Indeed, even when reverting a `CreateTableOp`, for example, `DropTableOp.from_table()` is used (see l. 967). But on principle I agree, we should not break it.

Reply:
we are on the edge of a new "minor" series 0.7 so it's a better time to change the API but I felt for the moment it's best to make this PR work without changing any public API, then make public API changes between these two objects something separate.